posthog/queries/funnels/funnel_event_query.py

```diff
@@ -115,17 +110,20 @@ def get_query(
         return query, self.params
 
     def _determine_should_join_distinct_ids(self) -> None:
-        if (
+        non_person_id_aggregation = (
```

Comment:
same as the other PR - should this be in every file separately?

Reply:
replied there but this is unrelated - I'm currently building this in with how the current system works, can think of separate refactors later.

note though that it's not _always_ the same, and having a way to account for that via an abstraction might end up being more complicated than a bit of repetition here and there